### DISCUSSION TITLE
useToggle hooks parameters should  only be an array or 2 parameters

### DISCUSSION THREAD

**Discussion by raflymln** [mantinedev/mantine#1421] [Feedback]
As seen on https://mantine.dev/hooks/use-toggle/, the parameters should only be an array and take the first array as the initial value. It helps the code more understandable and efficient.

Example:
```js
useToggle(['red', 'blue']) // "red" would be the default value
```
Or you could just accept two parameters instead of an array

Example:
```js
useToggle(1, 2) // 1 is the default, 2 is the second option
```

**Comment by rtivital** [comment]
Great idea, thanks for sharing, I'll consider it for next major release

**Comment by yverby** [comment]
How about adding the ability to pass more than two values in an array, or more than two parameters instead of an array for this hook?

For example (sort state), when more than two states are needed:

```js
useToggle([null, 'asc', 'desc'])
```

**Reply by raflymln** [reply]
Seems cool tho!
